packages/report-server/src/reportBuilder/transform/functions/index.ts

```diff
@@ -25,3 +34,19 @@ export const transformBuilders: Record<string, TransformBuilder> = {
   excludeRows: buildExcludeRows,
   insertRows: buildInsertRows,
 };
+
+export const transformSchemas: Record<
+  string,
+  {
+    type: string;
+    fields: Record<string, unknown>;
+  }
+> = {
+  insertColumns: insertColumnsParamsValidator.describe(),
+  excludeColumns: excludeColumnsParamsValidator.describe(),
+  updateColumns: updateColumnsParamsValidator.describe(),
+  mergeRows: mergeRowsParamsValidator.describe(),
```

Comment:
Hmmm so one thing I've noticed is that it seems `yup.describe()` returns `null` for any lazy validators. This is totally understandable, as the type of validator can't be known until a value is given, but does present a slight problem for us as we'd like to get some sort of schema for that validator here...

Ways we could address this might be:
- Re-writing those validators to not use lazy validation. Not too sure if this is possible though...
- Prodividing some schema definitions for the lazy validators to combine with the default `yup.describe()`?

Reply:
I've tried these two options and it didn't work. Another hacky way is to extend class `Lazy` and modify the `describe()`. More info is coming up from next comment.

Reply:
Thanks @rohan-bes for pushing me to do the schema! I did try before but I gave up..